src/Features/CSharp/Portable/ChangeSignature/CSharpChangeSignatureService.cs

```diff
@@ -258,6 +260,13 @@ private SyntaxNode GetNodeContainingTargetNode(SyntaxNode matchingNode)
                 return method.WithParameterList(method.ParameterList.WithParameters(updatedParameters).WithAdditionalAnnotations(changeSignatureFormattingAnnotation));
             }
 
```

Comment:
should local functions work with `UpdateParamTagsInLeadingTrivia` above?

Reply:
Good question. At this point, no. Local functions don't have xml docs.

Reply:
hmmm... i tried typing `///` and it created one for me. probably a bug then?

Reply:
Not really.  we're not going to stop you from writing the doc comment if you want.  After all, you may intend to move this to a class-method.  (or, it just may be you're missing a } and perhaps you meant this to be a class member).  I'm not sure we'd go out of our way to make this work.  but if it works, we'll keep allowing it to.

Reply:
actually i'm not able to reproduce it anymore